jboss-eap-clustered-rhel-VMSS/README.md

```diff
@@ -43,19 +43,19 @@ This uses RHEL 8.0 image which is a PAY AS YOU GO image and doesn't require the
 
 ## Prerequisites
 
-1. Azure Subscription with specified payment method (RHEL 8 is a market place product and requires payment method to be specified in Azure Subscription)
+1. Azure Subscription with specified payment method (RHEL 8.0 is a market place product and requires payment method to be specified in Azure Subscription)
 
 2. To deploy the template, you will need to:
 
-    - Choose an admin user name and password for your VM.  
+    - Choose an admin username and password for your VM.  
 
-    - Choose a EAP user name and password to enable the EAP manager UI and deployment method.
+    - Choose a EAP username and password to enable the EAP manager UI and deployment method.
```

Comment:
Modified this line as follows as per the feedback from other PRs for consistency. Please suggest

**JBoss EAP Username** and password